website/src/content/api-reference/custom-webpack-config.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: "Custom Webpack Config"
+---
+
+<Subtitle>
+The utilities allows you create a custom webpack configuration for a Merchant Center Application.
+</Subtitle> 
+
+# Installation
+
+```
+yarn add @commercetools-frontend/mc-scripts
+
+# or
+
+npm --save install @commercetools-frontend/mc-scripts
+```
+
+# Usage
+To create a custom webpack configuration for a Merchant Center application, you can use `createWebpackConfigForProduction` for production and `createWebpackConfigForDevelopment` for development.
+These are factory functions used to create a default webpack config. They both accept some optional arguments.
+
+## Production
+For production mode, you should create a `webpack.config.prod.(cjs, js)` file and use the function `createWebpackConfigForProduction` to create a default webpack config.
```

Comment:
```suggestion
## For production

For production mode, create a `webpack.config.prod.{js,cjs}` file and use the function `createWebpackConfigForProduction` to create and enhance the webpack configuration.

The Custom Application CLI will then use the configuration exported from this file instead of the default one.
```